src/commands/asset-compute/devtool.js

```diff
@@ -14,94 +14,18 @@
-    const addr = server.address();
-    const bind = typeof addr === 'string'
-        ? 'pipe ' + addr
-        : 'port ' + addr.port;
-    util.log('Listening on ' + bind);
-    const assetComputeDevToolUrl = `http://localhost:${addr.port}/?devToolToken=${randomString}`;
-    console.log('Asset Compute Developer Tool Server started on url', assetComputeDevToolUrl);
-    await open(assetComputeDevToolUrl);
-}
-
-/**
- * Normalize a port into a number, string, or false.
- */
-async function findOpenPort(preferredPort) {
-    return getPort({port: [preferredPort, preferredPort + 1, preferredPort + 2]});
-    // Will use specified port if available, otherwise fall back to a random port
-}
+require('dotenv').config;
 
 class DevToolCommand extends BaseCommand {
 
-    async run() {
+    async run(port) {
         const { flags } = this.parse(DevToolCommand);
-        const port = await findOpenPort(flags.port);
 
-        // random string for developer tool authorization token
-        let randomString;
-        try {
-            randomString = crypto.randomBytes(32).toString("hex");
-        } catch(e) {
-            console.log(e);
-            throw new Error('Error: Not enough accumulated entropy to generate cryptographically strong data.');
-        }
         return new Promise((resolve, reject) => {
-            app.set('port', port);
-            app.set('devToolToken', randomString);
-
-            // Create HTTP server.
-            util.log('Starting Asset Compute Developer Tool Server on port', port);
-            this.server = http.createServer(app);
-            const httpTerminator = createHttpTerminator({ server: this.server });
-
-            // Listen on provided port, on all network interfaces.
-            this.server.listen(port);
-            this.server.on('error', error => {
-                if (error.syscall !== 'listen') {
-                    return reject(error);
-                }
-
-                // handle specific listen errors with friendly messages
-                const bind = typeof port === 'string'
-                    ? 'Pipe ' + port
-                    : 'Port ' + port;
-                switch (error.code) {
-                case 'EACCES':
-                    util.logError(bind + ' requires elevated privileges');
-                    break;
-                case 'EADDRINUSE':
-                    util.logError(bind + ' is already in use');
-                    break;
-                }
-                return reject(error);
-            });
-            this.server.on('listening', () => onListening(this.server, randomString));
-            this.server.on('close', () => {
-                util.log("Asset Compute Developer Tool Server Stopped");
-            });
-            this.onProcessExit(async () => {
-                util.log("Stopping Asset Compute Developer Tool Server");
-                httpTerminator.terminate();
-            });
+            require('@adobe/asset-compute-devtool').start(port);
```

Comment:
Better to be consistent and have the import at the top, so that scanning the top of the file gives you an overview of all the modules that are imported:

```js
const devtool = require('@adobe/asset-compute-devtool');

// ...

devtool.start(port);
```